app/views/rentals/_form.html.erb

```diff
@@ -0,0 +1,76 @@
+
+<div class="form-group">
+  <%= label_tag :studio %><br />
+  <%= select_tag(:studio, options_for_select([["No", false], ["Yes", true]], @rental.studio ), :class => "form-control") %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :two_bed %><br />
+  <%= select_tag(:two_bed, options_for_select([["No", false], ["Yes", true]], @rental.two_bed ), :class => "form-control") %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :three_bed %><br />
+  <%= select_tag(:three_bed, options_for_select([["No", false], ["Yes", true]], @rental.three_bed ), :class => "form-control") %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :wheelchair_accessible %><br />
+  <%= select_tag(:wheelchair_accessible, options_for_select([["No", false], ["Yes", true]], @rental.wheelchair_accessible ), :class => "form-control") %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :car_owner %><br />
+  <%= select_tag(:car_owner, options_for_select([["No", false], ["Yes", true]], @rental.car_owner ), :class => "form-control") %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :pet_owner %><br />
+  <%= select_tag(:pet_owner, options_for_select([["No", false], ["Yes", true]], @rental.pet_owner ), :class => "form-control") %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :hear_of_property %><br />
+  <%= text_field_tag :hear_of_property, @rental.hear_of_property, :class => "form-control" %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :reason_for_move %><br />
+  <%= text_field_tag :reason_for_move, @rental.reason_for_move, :class => "form-control" %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :housing_situation %><br />
+  <%= text_field_tag :housing_situation, @rental.housing_situation, :class => "form-control" %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :employer_name %><br />
+  <%= text_field_tag :employer_name, @rental.employer_name, :class => "form-control" %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :employer_address %><br />
+  <%= text_field_tag :employer_address, @rental.employer_address, :class => "form-control" %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :employer_city %><br />
+  <%= text_field_tag :employer_city, @rental.employer_city, :class => "form-control" %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :employer_state %><br />
+  <%= select_tag(:employer_state, options_for_select(us_states, "IL"), :class => "form-control") %>
```

Comment:
Not really sure. I didn't add that but left as is. I suppose it is possible for someone to be employed in Indiana.